Transport/Get-EXOMTLReport.ps1

```diff
@@ -303,6 +303,10 @@ function Get-MTLStatistics {
 
 ### Main ###
 
+# Set InformationPreference to allow Write-Information to be displayed to the screen.
+$OriginalInformationPreference = $InformationPreference
+$InformationPreference = 'Continue'
```

Comment:
We should do this correctly by doing a `try` `finally` block, so this always gets set back correctly.